kudu/kudu.py

```diff
@@ -0,0 +1,73 @@
+import urllib2
+import json
+import argparse
+
+
+def main(servers):
+    metrics = []
+    for i in range(len(servers)):
+        server = servers[i]
```

Comment:
Wrap the logic within the for loop within a `try..except` block and log in case of errors? Currently an error related to polling a single server will cause the entire script to fail.